packages/angular/cli/commands/run.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/schema",
   "$id": "ng-cli://commands/run.json",
-  "description": "Runs a custom target defined in your project.",
+  "description": "Runs an architect target with a custom build configuration defined in your project.",
```

Comment:
Note sure about the terminology here, if it should be `build` or `builder`

```
custom build configuration -> custom builder configuration
```

Reply:
I would use `Builder` when it's not a build command, yes. So in this case yes, that's a good idea.

Reply:
Is `Builder` another tool or technical term that we need to introduce?